app/http.php

```diff
@@ -0,0 +1,353 @@
+<?php
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+use OpenRuntimes\Proxy\Health\Health;
+use OpenRuntimes\Proxy\Health\Node;
+use Swoole\Coroutine\Http\Client;
+use Utopia\Logger\Log;
+use Utopia\Logger\Logger;
+use Swoole\Coroutine\Http\Server;
+use Swoole\Http\Request as SwooleRequest;
+use Swoole\Http\Response as SwooleResponse;
+use Swoole\Runtime;
```

Comment:
```suggestion
use OpenRuntimes\Proxy\Health\Node;
use Utopia\Logger\Log;
use Utopia\Logger\Logger;
use Swoole\Coroutine\Http\Client;
use Swoole\Coroutine\Http\Server;
use Swoole\Http\Request as SwooleRequest;
use Swoole\Http\Response as SwooleResponse;
use Swoole\Runtime;
```

Reply:
Just reordered the list to have namespaces grouped.

Reply:
Reordered a little more to put groups together